package.json

```diff
@@ -34,6 +34,7 @@
     "@angular/platform-browser": "2.3.0",
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
```

Comment:
> so the more we can coalesce around a standard way of doing things that doesn't depend on specific forks of specific presets the better

Yeah true that makes sense. This is the same issue I'm finding with preset-20xx in general. It doesn't seem necessary to have it anymore since it's rare for someone to only want a specific year anyway. es2015, 16,17 presets aren't that useful anymore when you can use preset-latest or preset-env. 
Otherwise then there will be a 2018, etc and there's already squatted pkgs for some of these haha (unless we switch to `@babel/preset-es20xx`

So basically we could also make it an option under preset-es2015 and under preset-env if that is :+1:.